src/features/network/data/index.ts

```diff
@@ -270,24 +270,6 @@ export const networkConfigAtom = atom<NetworkConfigWithId>((get) => {
   }
   const config = networkConfigs[id]
 
-  // This clears a connected mnemonic wallet after a page reload, as the mnemonic is not stored and cannot be used to sign transactions.
```

Comment:
@neilcampbell I don't think we need this logic, MNEMONIC wallet is cleared automatically after a page reload. Is there any special cases that I should test?

Reply:
If that's the case we can likely removed. I'm curious if the stored state is cleared on reload by use-wallet.
3 test cases to run after removing this code:
- connect mnemonic wallet, reload, check you connect the same mnemonic wallet and sign/send a transaction
- connect mnemonic wallet, reload, check you connect a different mnemonic wallet and sign/send a transaction
- connect mnemonic wallet, reload, check you can connect a different wallet provider and sign/send a transaction

Reply:
I can confirm that after reloading, I can connect to a different mnemonic wallet and sign the transactions.